ui/app/AppLayouts/Profile/views/wallet/NetworksView.qml

```diff
@@ -13,7 +13,8 @@ Item {
     id: root
     signal goBack
 
-    property WalletStore walletStore
+    property var layer1Networks
```

Comment:
See here, what type is `layer1Networks`? A stringlist, comma separated string, JS Object? It's totally not obvious how to assign to it, whereas if it had stayed in the store, the logic at least stays at the same place

Reply:
the original is also using `var`, implying it's an object, I just kept it the same

Reply:
since this a view, dont we want to share the store directly instead?